### PR TITLE
Update migration.md with info about synchronization

### PR DESCRIPTION
**Pull request by ShapeShifter499**
Add information about synchronizing after migration. There could be issues if the server has been offline for a while and a user reconnects clients for synchronization.

### CODE REVIEW

**Comment by ShapeShifter499** [comment]
Sorry about the commits, everything should be okay now. Unless you had anything to add.

**Comment by ShapeShifter499** [comment]
Added missing "sign-off" message to git commit

**Comment by szaimen** [comment]
Hi, thanks for the PR! Maybe we should add a link at the end of each of the guides above? (Otherwise people will likely miss this?)

File: migration.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ The procedure for migrating only the files works like this:
 1. Next, run `sudo docker run --rm --volume nextcloud_aio_nextcloud_data:/mnt/ncdata:rw alpine chown -R 33:0 /mnt/ncdata/` and `sudo docker run --rm --volume nextcloud_aio_nextcloud_data:/mnt/ncdata:rw alpine chmod -R 750 /mnt/ncdata/` to apply the correct permissions. (Or if `NEXTCLOUD_DATADIR` was provided, apply `chown -R 33:0` and `chmod -R 750` to the chosen path.)
 1. Start the containers again and wait until all containers are running
 1. Run `sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ files:scan-app-data && sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ files:scan --all` in order to scan all files in the datadirectory.
+2. If the restored data is older than any clients you want to continue to sync, for example if the server was down for a period of time during migration, you may want to take a look at [Synchronising with clients after migration](/migration.md#synchronising-with-clients-after-migration) below.
```
Comment (szaimen) [inline migration.md:21]:
```suggestion
1. If the restored data is older than any clients you want to continue to sync, for example if the server was down for a period of time during migration, you may want to take a look at [Synchronising with clients after migration](/migration.md#synchronising-with-clients-after-migration) below.
```

**Comment by ShapeShifter499** [comment]
Added links and fixed that typo

**Review by szaimen — APPROVED** [review]
LGMT, thanks! :)

**Pull request merged**